app/controllers/concerns/error_handling.rb

```diff
@@ -28,7 +28,7 @@ def handle_access_denied(exception)
     end
 
     def handle_forgery_protection(exception)
-      System::ErrorReporting.report_error(exception)
+      System::ErrorReporting.report_error(exception) if params[:authenticity_token].present?
       handle_error(exception, :forbidden)
```

Comment:
This contains adding logs in Rails.logger :)

Reply:
Maybe only if the exception is of type `ActionController::InvalidAuthenticityToken` ?